simln-lib/README.md

```diff
@@ -0,0 +1,13 @@
+# sim-ln
+
+A Lightning Network simulation library that enables testing and analysis of payment routing behavior in a controlled environment.
```

Comment:
nit: in a controlled environment / on any lightning network.

We could theoretically run on signet where you don't control all the nodes.